thunder/core/jit_ext.py

```diff
@@ -915,6 +915,20 @@ def _general_jit_torch_autograd_function_apply_lookaside(obj: Any, *args, **kwar
     return _interpret_call(custom_forward, wrapped_ctx, *args_, **kwargs_)
 
 
+@general_jit_lookaside(torch.finfo)
+def _general_jit_torch_finfo_lookaside(obj: Any):
+    try:
+        thunder_dtype = unwrap(obj)
+        torch_dtype = thunder.dtypes.to_torch_dtype(thunder_dtype)
+        res = torch.finfo(torch_dtype)
+    except BaseException as e:
+        return do_raise(e)
+
+    pr = ProvenanceRecord(PseudoInst.OPAQUE, inputs=[wrap_const(torch.finfo).provenance])
```

Comment:
We should add obj.provenance here, too.
```suggestion
    pr = ProvenanceRecord(PseudoInst.OPAQUE, inputs=[wrap_const(torch.finfo).provenance, obj.provenance])
```

Reply:
Oh, I thought I had it.. will update this.